doc/telescope.txt

```diff
@@ -765,8 +765,8 @@ options you want to use. Here's an example with the live_grep picker:
 <
 
 builtin.live_grep({opts})                      *telescope.builtin.live_grep()*
-    Search for a string and get results live as you type (respecting
-    .gitignore)
+    Search for a string and get results live as you type, respecting
+    .gitignore
```

Comment:
vim doc is autogenerated, so needs to be addressed here: https://github.com/nvim-telescope/telescope.nvim/blob/master/lua/telescope/builtin/init.lua#L53